### DISCUSSION TITLE
[Keyball44] コンスルーを用いてビルドしたときに OLED と ProMicro 間の隙間が気になる

### DISCUSSION THREAD

**Discussion by kk-shinoda** [Yowkees/keyball#539] [Q&A]
## 背景
Keyball44 と別途 12P 2.5mm のコンスルーを購入し、[ビルドガイド](https://github.com/Yowkees/keyball/blob/main/keyball44/doc/rev1/buildguide_jp_sub.md#build-guide-for-keyball44) を参考に組み立てを行っています。

その際、OLED の高さ(ソケット+ピンヘッダ)に対して ProMicro(コンスルー+ProMicro の高さ) の高さが低いため支えてあげないと OLED が垂れてしまいます。

なにか解決方法があれば教えていただきたいです。

![IMG_5442](https://github.com/Yowkees/keyball/assets/86135975/e52feb63-d4e4-4fd1-991e-d21a485892ee)

**Comment by Ryo624** [comment]
12Pコンスルーを使用する場合、４Pピンソケットは背の低いものに変えないとProMicroとOLEDの間に隙間が出来てしまいます。
今後も12Pコンスルーの使用を予定している場合は高さが3.5ｍｍの４Ｐピンソケットに取り替えてください。

**Reply by kk-shinoda** [reply]
@Ryo624 さん
コメントありがとうございます。情報大変助かりました :pray:

本件クローズさせていただきます。
